README.md

```diff
@@ -12,3 +12,4 @@
 
 - [Linked List Insertions](javascript/code-challenges/linked-list/ll-insertions/README.md)
 
+- [Linked List Zip](javascript/code-challenges/llZip/README.md)
```

Comment:
What a wonderful whiteboard image! 
I would love to see some code and tests added to this!